src/Middlewares/Middleware.php

```diff
@@ -55,7 +55,7 @@ public function before(Request $request)
         {
             /** @var ConfigRepository $config */
             $config = pluginApp(ConfigRepository::class);
-            $enabledCurrencies = explode(', ',  $config->get('Ceres.currency.available_currencies') );
+            $enabledCurrencies = explode(', ',  $config->get($config->get('IO.template.template_plugin_name') . '.currency.available_currencies') );
```

Comment:
the changes you made are alright, thank you for that.
but can you please use the class TemplateConfigService from IO instead of
$config->get($config->get('IO.template.template_plugin_name') ?